php/Blocks/Controls/Checkbox.php

```diff
@@ -49,11 +49,10 @@ public function register_settings() {
 		$this->settings[] = new ControlSetting( $this->settings_config['help'] );
 		$this->settings[] = new ControlSetting(
 			[
-				'name'     => 'default',
-				'label'    => __( 'Default Value', 'genesis-custom-blocks' ),
-				'type'     => 'checkbox',
-				'default'  => '0',
-				'sanitize' => [ $this, 'sanitize_checkbox' ],
+				'name'    => 'default',
+				'label'   => __( 'Default Value', 'genesis-custom-blocks' ),
+				'type'    => 'checkbox',
+				'default' => false,
```

Comment:
The default should be a `boolean`, as a `Checkbox` field has a type of `boolean`